### DISCUSSION TITLE
Can't download base model or convert to onnx

### DISCUSSION THREAD

**Discussion by DaiMaouYTS** [azuritecoin/OnnxDiffusersUI#61] [Q&A]
everytime I run the command: `python convert_stable_diffusion_checkpoint_to_onnx.py --model_path="runwayml/stable-diffusion-v1-5" --output_path="model/stable_diffusion_onnx"`

or the command for converting a model right away, the installation progress bar goes to 2% and stops and I get this error:
`requests.exceptions.ConnectionError: HTTPSConnectionPool(host='cdn-lfs.huggingface.co', port=443): Read timed out.
`

I made sure everything was properly installed, hugging face also gave me "login successful" the only thing I couldn't do is the "accept the terms and conditions for the model" because that never appeared for me, but that shouldn't be causing a problem when converting a ckpt that is on my pc... one time the installation progress bar reached 40% and stopped, one time it reached 5%, everytime it stops at a random point

I'm new to all this so I don't really understand which one is supposed to be the error that is causing it, but there's different ones
`urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='cdn-lfs.huggingface.co', port=443): Read timed out.`
`TimeoutError: The read operation timed out`
<img width="1909" alt="image" src="https://user-images.githubusercontent.com/120621256/208259336-4f6dfa9a-4ca1-4a67-a8d6-26ee00d59675.png">

<img width="594" alt="image" src="https://user-images.githubusercontent.com/120827179/208261361-a1867d23-c6e9-443a-aaca-9ee86f10f6a9.png">

Here's my python and git and pip versions

**Comment by JJGall** [comment]
When you view the model on huggingface it will ask you to accept terms before you can access it. I don't know if that is your issue though. The script downloads it from the website and then converts it, is my understanding. If you haven't accepted the terms it may be messing that step up.

**Comment by azuritecoin** [comment]
Like @JJGall said, you need to go to the model webpage here: https://huggingface.co/runwayml/stable-diffusion-v1-5

On that page there's the terms and conditions that you need to accept.

**Reply by DaiMaouYTS** [reply]
the terms weren't appearing for me, but someone managed to fix it, they said it's trying to install the stable diffusion even if try other models, so they told me to add some files to something and it worked!

**Reply by GITwhy18** [reply]
what did you do to fix it?
